src/status_im2/contexts/chat/messages/content/deleted/view.cljs

```diff
@@ -26,10 +26,10 @@
     (i18n/label :t/deleted-this-message)]])
 
 (defn- compute-on-long-press-fn
-  [{:keys [deleted? pinned] :as message}
-   {:keys [message-pin-enabled] :as context}]
+  [{:keys [deleted?] :as message}
+   context]
   ;; only show drawer for user who has the permission to unpin messages
-  (when (and pinned deleted? message-pin-enabled)
```

Comment:
`(and pinned deleted? message-pin-enabled)` this is used to only allow users with permission to unpin message to see the long press menu to access to the unpin option

Seems `when-not deleted?` will lead to an empty long press menu for users with no unpin permission? Or is this checked or somehow changed somewhere else?

Reply:
You are probably right, I will double check that